File: school-register-app/src/people/dtos/person-dto.ts
import { SchoolIdAndNameDto } from "../../school/dtos/school-id-and-name-dto";
import { SettlementDto } from "../../settlement/dtos/settlement-dto";

export class PersonDto {
  id: number;
  firstName: string;
  lastName: string;
  uic: string;
  birthDate: Date;
  schoolId : number;
 
  birthPlace: SettlementDto;
}

File: SchoolRegisterApp.Models/Dtos/PersonDto.cs
using SchoolRegisterApp.Models.Entities;

namespace SchoolRegisterApp.Models.Dtos
{
    public class PersonDto
    {
        public int Id { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public string Uic { get; set; }

        public int? SchoolId { get; set; }

        public DateTime BirthDate { get; set; }

        public Settlement BirthPlace { get; set; }

    }
}
